intents.go

```diff
@@ -1,6 +1,7 @@
 package model
 
 import (
+	"encoding/json"
```

Comment:
"Do you want to use `github.com/goccy/go-json`? It is fully compatible and performs significantly better."

Reply:
indeed, done.